cmd/lnmuxd/run.go

```diff
@@ -126,6 +130,18 @@ func runAction(c *cli.Context) error {
 		return err
 	}
 
+	promAddress := cfg.PrometheusAddress
+	if promAddress == "" {
+		promAddress = DefaultPrometheusAddress
+	}
+
+	// Instantiate a new HTTP server and mux.
+	metrics := &http.Server{
```

Comment:
Looking ahead at https://github.com/bottlepay/lnmux/issues/43, we may need to use a different http server to allow the registration of multiple endpoints. But can be done later.

Reply:
I've changed the configuration option to be `InstrumentationAddress` and the related wording. I also changed the mux of the HTTP server to a fresh `ServeMux` and registered the Prometheus handler as `/metrics` so we can just add a `/pprof` handler for #43.